README.asciidoc

```diff
@@ -99,8 +100,8 @@ Building
 
 Kakoune dependencies are:
 
- * A C\+\+11 compliant compiler (GCC >= 5 or clang >= 3.4) along with its
-   associated C\+\+ standard library (libstdc\+\+ or libc\+\+)
+ * A {cpp}11 compliant compiler (GCC >= 5 or clang >= 3.4) along with its
```

Comment:
The readme is expected to be readable as text as well, so I'd rather keep `C++` written as-is in the source.